internal/config/types.go

```diff
@@ -158,9 +158,10 @@ func (p *Python) FillDefaults(
 }
 
 type R struct {
-	Version        string `toml:"version,omitempty" json:"version"`
-	PackageFile    string `toml:"package_file,omitempty" json:"packageFile"`
-	PackageManager string `toml:"package_manager,omitempty" json:"packageManager"`
+	Version          string `toml:"version,omitempty" json:"version"`
+	PackageFile      string `toml:"package_file,omitempty" json:"packageFile"`
+	PackageManager   string `toml:"package_manager,omitempty" json:"packageManager"`
+	RequiresRVersion string `toml:"requires_r_version,omitempty" json:"requiresRVersion"`
```

Comment:
Ultra nit: since we're already within an `R` struct, and golang tends to favor terse variable naming in general, what about just `Requires`?